install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -1,4 +1,17 @@
-You have successfully installed {{ title .Chart.Name }}.
+{{- if (and (.Values.preflight.enabled) (not (.Values.agent.enabled)) (not (.Values.config.enabled)) (not (.Values.operator.enabled))) }}
+    You have successfully ran the prefight check.
+    Now make sure to check the number of READY pods is the same as the number of running cilium pods.
+    Then make sure the cilium preflight deployment is also marked READY 1/1.
+    If you have an issues please refer to the CNP Validation section in the upgrade guide.
```

Comment:
I recognize that the user likely already has the upgrade docs up at this point so this should make sense without needing to add a direct link to the upgrade guide.